.github/workflows/test.yml

```diff
@@ -123,8 +123,8 @@ jobs:
   # CLIENT (without types test)
   #
   client:
-    timeout-minutes: 35
-    runs-on: ${{ matrix.os }}
+    timeout-minutes: ${{ github.event_name == 'schedule' && 100 || 35 }}
+    runs-on: ${{ github.event_name == 'schedule' && contains(matrix.os, 'buildjet') && 'ubuntu-latest' || matrix.os }}
```

Comment:
One downside with this approach is that the job names will still contain `buildjet-4vcpu-ubuntu-2004` as it came from the matrix, although on scheduled runs it will be run will the free `ubuntu-latest`. I could also replace these in the names, but thought the complexity is enough. Agree?

Reply:
You can see which runner is active at the beginning of the job https://github.com/prisma/prisma/actions/runs/4834439592/jobs/8615666902?pr=19008#step:1:10

Reply:
Also, I doubled doubled the timeouts when running on schedule.